app/controllers/security_check_failed_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class SecurityCheckFailedController < ApplicationController
```

Comment:
"Security check" could be pretty ambiguous in the broader application, and maybe we want to qualify the name / module to refer to its applying to sign-in?